### ISSUE TITLE
Gutter wraps under container when client resizes the browser

### ISSUE THREAD

**Issue by AdamMikacich** [nathancahill/Split.js#138]
![splitjsbug](https://user-images.githubusercontent.com/20669161/41329356-2044ba92-6e82-11e8-82df-dc67385dcb86.gif)
**Steps to reproduce bug:**
1. (optional: reset sizes)
2. Drag a gutter fully to the right or collapse the gutter.
3. Minimize the browser by resizing it to the left.

**Bug**
The gutter on the right wraps under the container instead of staying at 100% within the container.

**Question**
Is there a current solution to prevent the gutters from wrapping outside of the container?

**Comment by burtonator** [comment]
Yeah.. I'm having the same issue in my app.. Completely breaks everything.. the gutter forces itself down below the main area. It has to do with float: left on the butter but I can't figure out why it's triggered.

**Comment by burtonator** [comment]
A potential work around seems to be to use display: table and display: table-cell for the sidebars BUT calc() with percentages minus a px width doesn't work.

It DOES work properly with only percentages though. It also seems to properly factor in the width of the gutter.  Just don't know if it works on other browsers.

**Comment by burtonator** [comment]
ok... this is a work around...

create the splitter like this:

```
        elementStyle: (dimension: 'width' | 'height', elementSize: number, gutterSize: number) => {
            return {
                width: `${elementSize}%`
            };
        },
```

then use display: table on the parent and display: table-cell on all the boxes.  Seems to work fine. I'll report more when I get it in production.

**Comment by nathancahill** [comment]
Yes, this is fixed.
